lib/timeline/timeline.dart

```diff
@@ -224,9 +229,9 @@ class TimelineScreen extends Screen {
     for (TimelineEvent event in events) {
       final TimelineThread thread =
           new TimelineThread(timelineData, event.args['name'], event.threadId);
-      if (thread.isVisible) {
-        timelineData.addThread(thread);
-      }
+      // TODO: consider pruning which threads we add based on which ones are
```

Comment:
isVisible checked that the thread name matched 'io.flutter', and none of the threads matched that name. I talked to Devon about this and he thinks the thread names reported by the VM probably changed. Removing this might make more threads visible than we need, but we can prune later.

Reply:
sg